### ISSUE TITLE
I want to use Google Map geometry library. With @google/maps NPM package. 

### ISSUE THREAD

**Issue by virtualshailesh** [googlemaps/google-maps-services-js#216]
I want to see if a location is within the range or not of **overview_polyline **. 
I can do that on Client site on HTML.

```
  var request = {
        origin: document.getElementById("from").value,
        destination: document.getElementById("to").value,
        travelMode: google.maps.DirectionsTravelMode.DRIVING
      }

      // Make the directions request
      directionService.route(request, function (result, status) {
        console.log(result);

        if (status == google.maps.DirectionsStatus.OK) {
          // directionsRenderer.setDirections(result);

          // Box around the overview path of the first route
          var path = result.routes[0].overview_path;
          console.log(result.routes[0]);

          var boxes = routeBoxer.box(path, distance);
          var polyArr = [];
          boxes.forEach(function (element) {
            let top_latlng = {
              lat: element['ma'].l,
              lng: element['fa'].l
            };
            let bottom_latlng = {
              lat: element['ma'].j,
              lng: element['fa'].j
            };
            polyArr.push(top_latlng);
            polyArr.push(bottom_latlng);
          });
          // console.log(polyArr);

          
          drawBoxes(boxes);

          var point = new google.maps.LatLng(28.566854, 77.341518);
          var path_poly = new google.maps.Polyline({
            path: polyArr
          });

          if (google.maps.geometry.poly.isLocationOnEdge(point, path_poly, 0.01)) {
            alert('fall');
          } else {
            alert('not fall');
          }


        } else {
          alert("Directions query failed: " + status);
        }
      });
    }
```

But I am not able to do the same in node js.
I have installed the @google/maps NPM package it is also not providing the **overview_path**.

What I want is, If I can do something like this on node as well using **overview_polyline** instead of **overview_path**. 

MY code on node

```
const googleMapsClient = require('@google/maps').createClient({
    key: '<API_KEY>',
    Promise: Promise
  });

googleMapsClient.directions({origin: '<START POINT>', destination: '<END POINT>', mode: 'driving'})
.asPromise()
.then((response) => { 

// I want to use it here....
  res.json(response.json)
})
.catch((err) => {
  console.log(err);
  res.json(err);
});
});
```

Thanks

**Comment by Luis-Palacios** [comment]
@virtualshailesh have you find a way around this? I'm having the same problem I need access to the overview_path property but is not available  on the response object 😞

**Comment by Luis-Palacios** [comment]
I've actually found a workaround.

TLDR;
1. `npm install @mapbox/polyline`
2. `const polyline = require('@mapbox/polyline');`
3.  Convert overview_polyline to overview_path
```js
... your callback here
(err, res) => {
const encodedPolylinePoints = res.json.routes[0].overview_polyline.points;
const overviewPath = polyline.decode(encodedPolylinePoints);
...
}
```

Detailed
Using this library [polyline](https://github.com/mapbox/polyline) and reading this [post](https://benkammerling.wordpress.com/2012/02/05/google-maps-overview_polyline-path-decode/#comment-185) help me understand the relationship between the overview_polyline and overview_path so you can actually decode the polyline points and generate and overview_path the same as the javascript API direction service does
